presto-product-tests/bin/run_on_docker.sh

```diff
@@ -22,12 +22,8 @@ function retry() {
   return ${EXIT_CODE}
 }
 
-function hadoop_master_container(){
-  environment_compose ps -q hadoop-master
```

Comment:
See https://github.com/prestosql/presto/blob/bec0121e1ea4620ecb65effdbce96d688ad051be/presto-hive-hadoop2/bin/common.sh#L21

Reply:
Please revert. I use this function :)